migtests/scripts/functions.sh

```diff
@@ -799,4 +797,63 @@ move_tables() {
     # Update the JSON file with the new lists of sharded and colocated tables
     jq --indent 4 --argjson new_sharded_tables "$new_sharded_tables_json" --argjson remaining_colocated_tables "$remaining_colocated_tables_json" \
        '.Sizing.SizingRecommendation.ShardedTables += $new_sharded_tables | .Sizing.SizingRecommendation.ColocatedTables = $remaining_colocated_tables' "$json_file" > tmp.json && mv tmp.json "$json_file"
-}
\ No newline at end of file
+}
+
+normalize_json() {
+    local input_file="$1"
+    local output_file="$2"
+
+    jq 'walk(
+        if type == "object" then
+            if has("ObjectNames") and (."ObjectNames" | type == "string") then
+                .ObjectNames |= (split(", ") | sort | join(", "))
+            else
+                .
```

Comment:
what does this else . means here?
Can't we just have condiitons with `if`, `else if`,....

Reply:
The . in jq represents the current element being processed.

In this context, if condition then action else . end means "if the condition is true, perform the action; otherwise, do nothing and leave the element unchanged."

Let me see where all I can change it to if, else if. It should be possible.